assets/js/modules/analytics/components/module/ModuleOverviewWidget/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * ModuleOverviewWidget component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { DATE_RANGE_OFFSET, MODULES_ANALYTICS } from '../../../datastore/constants';
+import Header from './Header';
+import Overview from './Overview';
+import SiteStats from './SiteStats';
+const { useSelect } = Data;
+
+export default function ModuleOverviewWidget( { Widget, WidgetReportError } ) {
+	const [ selectedStat, setSelectedState ] = useState( 0 );
+
+	const dates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+
+	const args = {
+		...dates,
+		dimensions: 'ga:date',
+		metrics: [
+			'ga:users',
+			'ga:sessions',
+			'ga:bounceRate',
+			'ga:avgSessionDuration',
+		],
+	};
```

Comment:
While I suggested to combine the metrics into one request, now while testing I noticed a regression over what the current/legacy overview widget displays, which is due to a really weird issue with the Analytics API itself 😞 

We'll need to request the `ga:users` metric total separately, because when adding the `ga:date` dimension, it is different (for whatever inexplicable reason, this has come up before, and removing the dimension is the way to fix/work around it). I just re-tested this again now, it really is only the `ga:users` metric where the totals differ based on whether the dimension is there or not. The other three metrics have the same totals between both requests.

I suggest we make two requests here, the one you have, and another one that is exactly the same, just without the `dimensions` argument. Then, `Overview` can use the one without the dimensions to have the correct number.

Reply:
Sounds good. Updated.